pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -118,8 +118,15 @@ type GrafanaSpec struct {
 
 	LogLevel string      `json:"logLevel,omitempty"`
 	Service  ServiceSpec `json:"service,omitempty"`
-	Username string      `json:"username,omitempty"`
-	Password string      `json:"password,omitempty"`
+
+	// +optional
+	// if username is omitted, Grafana will use `admin` as its username by default.
+	Username *corev1.SecretKeySelector `json:"username,omitempty"`
+
+	// +optional
+	// if password is omitted, Grafana will use `admin` as its password by default.
+	Password *corev1.SecretKeySelector `json:"password,omitempty"`
```

Comment:
Does it be Forward compatible to the previous TidbMonitor object? What about using a new property to indicate the grafana's password secretRef and make the current Password as Deprecated?

Reply:
Great, I think that makes sense. Will update this week.